Activities_BridgePath.py

```diff
@@ -0,0 +1,55 @@
+from Messages import Message_
+
+class Activities_Bridge_FP_TO_CBR_:
+    def __init__(self, V):
+        self.V = V
+        self.Status_Bridge = ""
+
+        # -------------------------------------------------- Creating Place
+        self.Place_Bridge = self.V.Place_Bridge
+        TF = self.Place_Bridge.CreatePlace()
+        if TF!=True:
+            self.V.Display.Menu_Quit()
+
+        # -------------------------------------------------- Characters
+        self.Char_Bob = self.V.Char_Bob
+
+        # -------------------------------------------------- Sounds
+        self.SEffect_Danger1 = self.V.SEffect_Danger1
+
+        # -------------------------------------------------- Camera Settings
+        TF = self.V.Display.FadeOut()
+        TF = self.V.Set.CameraFocus(self.Place_Bridge.SouthEnd)
+        TF = self.V.Set.CameraMode('track')
+        TF = self.V.Set.CameraBlend(4)
+        self.Status_Bridge = " --> Bridge_Setup "
+
+    def SC_Start(self):
+        TF = self.V.Set.DisableInput()
+        TF = self.V.Display.FadeIn()
+        TF = self.SEffect_Danger1.Play()
+        TF = self.Char_Bob.Enter(self.Place_Bridge.SouthEnd)
+        TF = self.V.Set.CameraFocus(self.Char_Bob.Name)
+        TF = self.Char_Bob.Face(self.Place_Bridge.NorthSign)
+
+        TF = self.Char_Bob.WalkTo(self.Place_Bridge.WestEnd)
+        TF = self.Char_Bob.WalkTo(self.Place_Bridge.Flowers)
+        _ = Message_(["Wow such a beautiful Flower. I will take it for my Love. She will be Happy. "], self.V.Action).ShowNarration()
+
+        TF = self.Char_Bob.WalkTo(self.Place_Bridge.NorthEnd)

```

Comment:
Here, the player is making transition automatically without user input you can write a function like below.
    def UC_Select_Task(self):
        self.V.Set.EnableInput()
        self.SEffect_Danger1.Play()
        while True:
            self.camelotMessage = input()
            if ("Talk" in self.camelotMessage) and (self.Char_Amelia_S.Name in self.camelotMessage):
                self.SC_Task01_Bob_Amelia_Talk()